### ISSUE TITLE
fullscreen

### ISSUE THREAD

**Issue by spiraloid** [f3d-app/f3d#1061]
I wish I could hit double click/enter and toggle fullscreen

**Comment by Meakk** [comment]
We used to have the option but removed it because it was causing conflicts with the OS on some platform.  
We realized it was the window manager responsibility, not ours.  
On Windows, pressing F11 should work.

**Comment by mwestphal** [comment]
See this related issue: https://github.com/f3d-app/f3d/issues/12

I agree with @Meakk , we don't really want to bring it back for now.

Could you explain what you are trying to achieve ?

**Comment by mwestphal** [comment]
@spiraloid please provide more info

**Comment by spiraloid** [comment]
I use f3d the same way I use irfanview.  As a viewer to inspect hundreds of media files in a directory. For 2D images,I double click on an image in a folder, it opens fullscreen, borderless and fit to the whole monitor without distraction.  Spacebar to advance, backspace to go back, esc to quit. Delete to delete file.  Fast Art review. 

Same process in f3d is double click .glb, click maximize.  Try to ignore titlebar, hit down arrow to load all images in folder. Right and left arrow. Clumsy.

**Comment by mwestphal** [comment]
Then adding fullscreen back is needed in order to avoid the title bar I suppose.

> it down arrow to load all images in folder. 

So you would like this to be automatic, right ? Being able to load a folder whenever you open a file.

**Comment by Meakk** [comment]
Extending the shell extension to have a "Open folder with f3d" in the right click context menu would make sense to me

**Comment by mwestphal** [comment]
TBH I was more thinking of a F3D option, not a OS specific solution.

**Comment by mwestphal** [comment]
Should we open an issue for folder opening integration @Meakk ?

**Comment by Meakk** [comment]
Yes, I guess it's an interesting feature we should implement at least on Windows.

**Comment by mwestphal** [comment]
Added here: https://github.com/f3d-app/f3d/issues/21
